govtool/frontend/src/hooks/forms/useVoteContextForm.tsx

```diff
@@ -35,20 +43,29 @@ export const useVoteContextForm = (
 
   const generateMetadata = useCallback(async () => {
     const { voteContextText } = getValues();
-
-    const canonizedJsonHash = blake2bHex(voteContextText, undefined, 32);
+    const body = generateMetadataBody({
+      data: {
+        comment: voteContextText,
+      },
+      acceptedKeys: ["comment"],
+      standardReference: CIP_100,
+    });
+    const jsonld = await generateJsonld(body, CIP_100_CONTEXT, CIP_100);
+
+    const canonizedJson = await canonizeJSON(jsonld);
```

Comment:
The need for canonising was removed

Reply:
Should we get rid of all the canonized jsons then (GA and DRep metadatas)?

Reply:
yes, for all metadata

Reply:
I apologise, this must have slipped through the cracks in our communications